.github/workflows/ubuntu-windows.yml

```diff
@@ -0,0 +1,363 @@
+name: Windows-Ubuntu
+
+on:
+  push:
+    branches:
+      - main
+      - feature/*
+  release:
+    types: [ created ]
+
+env:
+  GITHUB_TOKEN: ${{ github.token }}
+
+jobs:
+
+  build:
+    name: ${{ matrix.os }} shared=${{ matrix.shared }} sirius=${{ matrix.sirius }} xprs=${{ matrix.xprs }}
+    runs-on: ${{ matrix.os }}
+    env:
+      Sirius_Ref: antares_integration
+      XPRS_Path: ${{ github.workspace }}/xpressmp813
+      XPRESS: ${{ github.workspace }}/xpressmp813/bin
+      XPRS_LIB_Path: ${{ github.workspace }}/xpressmp813/lib
+    strategy:
+      matrix:
+        os: [windows-2019, ubuntu-20.04]
+        xprs: [ON, OFF]
+        sirius: [ON, OFF]
+        shared: [ON, OFF]
+        extras: [ON, OFF]
+
+        exclude:
+          - xprs: OFF
+            sirius: OFF
+          - os: windows-2019
+            shared: ON
+          - os: windows-2019
+            shared: OFF
+            extras: OFF
+          - os: ubuntu-20.04
+            shared: OFF
+            extras: ON
+          - os: ubuntu-20.04
+            shared: ON
+            extras: OFF
+        include:
+          - extras: ON
+            java: ON
+            python: ON
+            dotnet: OFF
+          - extras: OFF
+            java: OFF
+            python: OFF
+            dotnet: OFF
+
+
+    steps:
+      - name: set name variables
+        id: names
+        shell: bash
+        run: |
+          REF=${GITHUB_REF#refs/*/}
```

Comment:
I want to retrieve only the branch name or the tag name wihtout `refs/heads/` or other strings preceding the branch name.
we might simply abandon this bit since i find not essential to have the tagname or the branchname in the artifact filename.

Whant do you think @pet-mit  and @flomnes ? should we name the artifacts with the addition of the branchname or the tagname?
`ortools_install_windows-2019_static_sirius.zip` 
or
 `ortools_install_windows-2019_static_sirius_tagname.zip`  `ortools_install_windows-2019_static_sirius_branchname.zip`

Reply:
removed